src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <div className="Person">
+      <p className="Person__name">
+        {`My name is ${name}`}
+      </p>
+      {age !== undefined && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${partner}`}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
Use the ternary operator for the text only, the `p` tag should be outside of it.

By the way, do this instead of template literal
```suggestion
         {partnerName} is my ${partner}
```